### PR TITLE
Moved the self.draw_block call from __init__ to resize_to_fit. Should fix #55

### LINKED ISSUE #55: 'I' block is not displayed in small_board
We brought a bug to main:(

'I' block is not displayed in small_board if it's the first next_block (i.e. if resize_to_fit is not called from inside the class).

### PR DESCRIPTION
**Pull request by Arrinao**
fix #55

### CODE REVIEW

**Comment by Arrinao** [comment]
How come 'fix #55' in the description doesn't register?

**Comment by Akuli** [comment]
You put it to the title, not to the description

File: tetris.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 PINK = "#FF00FF"
 TEAL = "paleturquoise3"
 
-block_letters = ["I", "L", "L_rev", "O", "E", "Z", "Z_rev"]
+block_letters = ["I"]
```
Comment (Akuli) [inline tetris.py:24]: We can't put this to main. Sorry.

File: tetris.py [hunk 2]
```diff
@@ -88,9 +88,11 @@ def run_gui():
         None,
         None,
     )
-
     small_board.resize_to_fit()
 
+    print(small_board.block_letter)
+    print(small_board.current_block_center)
```
Comment (Akuli) [inline tetris.py:94]: Delete these

File: tetris.py [hunk 3]
```diff
@@ -229,6 +230,7 @@ def resize_to_fit(self):
             self.current_block_center = (2, 0)
         else:
             self.current_block_center = (1, 1)
+        self.draw_block()
```
Comment (Akuli) [inline tetris.py:233]: This is confusing. I really wouldn't expect a function named `resize_to_fit` to draw anything to the canvas.

**Review by Akuli — CHANGES_REQUESTED** [review]

**Review by Akuli — APPROVED** [review]
Includes some changes unrelated to fixing #55 but whatevs

**Pull request merged**